docs/pages/database-access/rbac/configuring-auto-user-provisioning.mdx

```diff
@@ -0,0 +1,166 @@
+---
+title: Database Automatic User Provisioning (Preview)
+description: Configure automatic user provisioning for Database Access.
+---
+
+<Details
+  title="Version warning"
+  opened={true}
+  scope={["oss", "enterprise"]}
+  scopeOnly={true}
+  min="13.1"
+>
+  Automatic user provisioning for Database Access is available starting from
+  Teleport 13.1.
+</Details>
+
+Teleport can automatically create users in your database, removing the need for
+having to create each individual user account in advance, or using the same set
+of shared database accounts for all users.
+
+<Admonition type="note" title="Supported databases">
+Currently, automatic user provisioning is only supported for self-hosted and
+RDS PostgreSQL databases.
+</Admonition>
+
+## Prerequisites
+
+- Teleport cluster with a configured [self-hosted](../guides/postgres-self-hosted.mdx)
+  or [RDS](../guides/rds.mdx) PostgreSQL database.
+- Ability to connect to and create user accounts in the target database.
+
+## Step 1/3. Configure database admin
+
+Teleport should be able to connect to the database as a user that can create
+other users and assign them roles. We recommend creating a separate user
+designated specifically for Teleport automatic user provisioning. Let's call it
+`teleport-admin`.
+
+Teleport will use the same authentication mechanism when connecting as an admin
+user as for regular user connections: X.509 for self-hosted databases and AWS
+IAM for RDS. The admin user must have privileges within the database to create
+users and grant them privileges.
+
+<Tabs>
+<TabItem label="RDS PostgreSQL">
+RDS PostgreSQL admin user must have `rds_iam` role attached to allow IAM
+authentication as described in the [RDS guide](../guides/rds.mdx#step-56-create-a-database-iam-user).
+```sql
+CREATE USER "teleport-admin" login createrole;
```

Comment:
Seems like docs linter complains about unknown `createrole`  word so our dictionary needs to be extended: [cspell](https://github.com/gravitational/teleport/blob/master/docs/cspell.json )